build-system/tasks/check-sourcemaps.js

```diff
@@ -154,12 +153,15 @@ function checkSourcemapMappings(sourcemapJson, map) {
 
 /**
  * @param {string} map The map filepath to check
+ * @param {boolean} checkMappings Whether to test a mapping points to a correct source.
  */
-function checkSourceMap(map) {
+function checkSourceMap(map, checkMappings) {
```

Comment:
teeny should likely ignore nit: consider creating an obj with key=filename, and value=firstLineMapping info. then checkMapping would be unnecessary.